docs/configuration.md

```diff
@@ -169,5 +169,108 @@ php artisan up
 ```
 > {tip} You may customize the default maintenance mode template by defining your own template at `resources/views/errors/503.blade.php`.
 
+## Configuration options
+
+|Name|Description|Default|
+|---|---|---|
+|`APP_NAME`|The gallery name|`Lychee`|
+|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
+|`APP_DEBUG`|Enable debug mode|`false`|
+|`APP_URL`|The URL of your gallery|`http://localhost`|
+|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
+|`TIMEZONE`|The timezone of your server|system timezone of server|
+|`DEBUGBAR_ENABLED`|Enable debugbar (only for debugging)|`false`|
+|`LOG_CHANNEL`|Default log channel, `single`, `daily`, `slack`, `syslog`, `errorlog`, `monolog`, `custom` or `stack`|`stack`|
+|`DB_CONNECTION`|Database type, `mysql`, `pgsql`, `sqlite` or `sqlsrv` (`sqlsrv` is not supported officially and may not work)|`mysql`|
+|`DB_DATABASE`|Path to SQLite database or name of MySQL/PostgreSQL DB|`database/database.sqlite` or `forge`|
+|`DB_HOST`|Host of DB server|`127.0.0.1`|
+|`DB_PORT`|Post of DB server|`3306` (MySQL) or `5432` (PostgreSQL)|
+|`DB_USERNAME`|Username of database|`forge`|
+|`DB_PASSWORD`|Password of database user|empty string|
+|`DB_OLD_LYCHEE_PREFIX`|Table prefix of old Lychee database|empty string|
+|`DB_LOG_SQL`|Log SQL statements (only use for debugging)|`false`|
+|`BROADCAST_DRIVER`|The broadcaster used if the framework needs to broadcast an event, `pusher`, `redis`, `log` or `null`|`null`|
```

Comment:
Do we use this for anything? If not, then I suppose we could remove this one as well as `PUSHER_*` and `REDIS_*` variables below?

Reply:
The question was not whether Redis can be used by some other services, but if we use the broadcasting services. We don't.